wandb/sdk/data_types/object_3d.py

```diff
@@ -255,6 +264,17 @@ def from_point_cloud(
         point_cloud_type: "PointCloudType" = "lidar/beta",
         # camera: Optional[Camera] = None,
     ) -> "Object3D":
+        """
+        Initializes Object3D from a python object.
+
+        Arguments:
+            points (Sequence["Point"]): The points in the point cloud.
+            boxes (Sequence["Box3D"]): 3D bounding boxes intended to allow labeling parts of the point cloud. These
+                will be displayed in the point cloud visualization.
+            vectors (Optional[Sequence["Vector3D"]]): Each vector will be displayed in the point cloud
+                visualization. Can be used to indicate directionality of bounding boxes. Defaults to None.
```

Comment:
Nit: change verbage to active voice

Vectors (Optional[Sequence["Vector3D"]]): Each vector is displayed in the point cloud
                visualization. Can be used to indicate directionality of bounding boxes. Defaults to `None`.